src/client/App.jsx

```diff
@@ -62,10 +62,20 @@ const App = () => {
             />
             <LeftMenu />
             <Routes>
-                <Route path={INT_LINK.HOME} element={<Home />} />
+                <Route 
+                    path={INT_LINK.HOME} 
```

Comment:
What does INT mean here? I'm hazarding a guess that it's INTERNAL, if it is I'd recommend being more explicit. Think of the poor junior dev who's going to maintain this in 5 years ;)